webgui-new/src/server.ts

```diff
@@ -16,9 +16,10 @@ const main = async () => {
   const proxyHandler = createProxyMiddleware({
     target: process.env.BACKEND_URL,
     changeOrigin: true,
-    pathFilter: '**/*Service',
+    pathFilter: ['**/*Service', '**/*ga.txt'],
     pathRewrite: {
       '^/[^/]+/(.+Service)$': '/$1',
+      '^/ga.txt' : '/$1'
```

Comment:
This will not work, as there are no capture groups in `^/ga.txt`, so `$1` is an invalid backreference. It should be either `'^/(ga.txt)': '/$1'`, or simply `'^/ga.txt' : '/ga.txt'`.

Reply:
Strange. If this is not right, then the frontend should not be able to fetch the content of this file at all.

Reply:
Nevermind, after removing my local cache, I can see it is indeed wrong. Just makes me wonder why I could test the functionality with this in the first place. Fixed.

Reply:
Yes, its indeed strange that it worked, maybe some caching issue based on your previous approach.